homeassistant/components/homematicip_cloud/hap.py

```diff
@@ -137,6 +138,18 @@ def async_update(self, *args, **kwargs):
 
             self.home.update_home_only(args[0])
 
+    @callback
+    def async_create_entity(self, *args, **kwargs):
+        """Create a device or a group."""
+        is_device = EventType(kwargs["event_type"]) == EventType.DEVICE_ADDED
+        self.hass.async_create_task(self.async_create_entity_lazy(is_device))
+
+    async def async_create_entity_lazy(self, is_device=True):
+        """Delay entity creation to allow the user to enter a device name."""
+        if is_device:
+            await asyncio.sleep(30)
```

Comment:
Is there no callback when the user is done?

Reply:
No, there is no additional callback.

Reply:
Ok.